src/Contao/CsrfUrlBuilder.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * This file is part of contao-community-alliance/url-builder.
+ *
+ * (c) 2017 Contao Community Alliance.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * This project is provided in good faith and hope to be usable by anyone.
+ *
+ * @package    contao-community-alliance/url-builder
+ * @author     Sven Baumann <baumann.sv@gmail.com>
+ * @copyright  2014-2017 Contao Community Alliance.
+ * @license    https://github.com/contao-community-alliance/url-builder/blob/master/LICENSE LGPL-3.0
+ * @filesource
+ */
+
+namespace ContaoCommunityAlliance\UrlBuilder\Contao;
+
+use ContaoCommunityAlliance\UrlBuilder\UrlBuilder;
+use Symfony\Component\DependencyInjection\ResettableContainerInterface;
+
+/**
+ * URL builder with security token.
+ *
+ * @package ContaoCommunityAlliance\UrlBuilder\Contao
+ */
+class CsrfUrlBuilder extends UrlBuilder
+{
+    /**
+     * The service container.
+     *
+     * @var ResettableContainerInterface
+     */
+    protected $container;
+
+    /**
+     * CsrfUrlBuilder constructor.
+     *
+     * @param ResettableContainerInterface $container The service container.
+     *
+     * @param string                       $url       The url.
+     */
+    public function __construct(ResettableContainerInterface $container, $url = '')
```

Comment:
Why `Symfony\Component\DependencyInjection\ResettableContainerInterface` instead of `Symfony\Component\DependencyInjection\ContainerInterface`?

We never call the reset function nor do we require it anywhere.